### PR TITLE
Reply to email

### PR DESCRIPTION
**Pull request by nilesh-more**
Pull Request for Issue # .

### CODE REVIEW

**Comment by brianteeman** [comment]
Please describe the issue that this PR is intended to resolve and how it can be tested. Otherwise it will be closed.

**Comment by brianteeman** [comment]
In the absence of any description or test instructions I am closing this - it can always be re-opened when updated<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11015">issues.joomla.org/joomla-cms/11015</a>.</sub>

**Comment by nilesh-more** [comment]
**1. Summary of Changes**

In most of the Joomla e-commerce site we send an email to end user but in case end user wants to give a reply on received email then by default user can reply only to the email address from where it has received the email.Now by the below functionality user can reply to the email set in the Joomla backend in global configuration with a field as "Reply To".

**2. Testing Instructions**

1. Login into Administrator,
2. Go to Global configuration,
3. Go to Mail Settings,
4. Configure mail settings with newly added Reply To and Reply To Name fields,
5. Save all configured settings,
6. Send one email from Joomla website to end user(eg. abc@gmail.com) or any email id,
7. Open abc@gmail.com Gmail account
8. Open a mail which I/we sent recently,
9. Click on a reply button and check/verify reply to email address.

**3. Snap shots**

**3.1. Configure backend mail settings**
replyto

**3.2. Open received email from Joomla site admin**
gmail

File: installation/configuration.php-dist [hunk 1]
```diff
@@ -60,14 +60,16 @@ class JConfig {
...
-	public $fromname = '';
-	public $sendmail = '/usr/sbin/sendmail';
-	public $smtpauth = '0';
-	public $smtpuser = '';
-	public $smtppass = '';
-	public $smtphost = 'localhost';
+	public $mailer      = 'mail';
+	public $mailfrom    = '';
+	public $fromname    = '';
+	public $replyTo     = '';
```
Comment (brianteeman) [inline installation/configuration.php-dist]: shouldnt this be $replyto not $replyTo ??

**Comment by hardikPTailored** [comment]
Yes, i have check this issue successfully.
when i received email and i have click to replay button receiver email to set in replay email.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11015">issues.joomla.org/joomla-cms/11015</a>.</sub>

**Comment by hardiktailored** [comment]
I have not tested this item.<br /><br />Reply-To email works as we set in Mail Settings of Joomla website back-end Global Configuration.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11015">issues.joomla.org/joomla-cms/11015</a>.</sub>

**Comment by hardikPTailored** [comment]
I have tested this item :white_check_mark: successfully on 4f27c96a62e3328682a4bfb278d24176af417d85<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11015">issues.joomla.org/joomla-cms/11015</a>.</sub>

**Comment by hardiktailored** [comment]
I have tested this item :white_check_mark: successfully on 4f27c96a62e3328682a4bfb278d24176af417d85<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11015">issues.joomla.org/joomla-cms/11015</a>.</sub>

**Comment by brianteeman** [comment]
1. These two fields must be hidden when Send Mail is set to No

2. Can you change the string reply to => reply to email<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11015">issues.joomla.org/joomla-cms/11015</a>.</sub>

**Comment by coolbung** [comment]
@nilesh-more This needs a quick update based on Brian's comment.

**Comment by zero-24** [comment]
Back to pending. Thanks.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11015">issues.joomla.org/joomla-cms/11015</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 4b1d822f202b745f8c0b0d48648848739e760cd1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11015">issues.joomla.org/joomla-cms/11015</a>.</sub>

**Comment by nilesh-more** [comment]
@zero-24 Didn't get you?

**Comment by Sieger66** [comment]
I have tested this item :white_check_mark: successfully on 4b1d822f202b745f8c0b0d48648848739e760cd1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11015">issues.joomla.org/joomla-cms/11015</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11015">issues.joomla.org/joomla-cms/11015</a>.</sub>

**Pull request merged**